.github/actions/benchmarks/action.yml

```diff
@@ -4,8 +4,8 @@ runs:
   using: "composite"
   steps:
     - run: |
-        bazel run -c opt src/cc/lib/benchmark:grpc_benchmark --config=linux
-        bazel run -c opt src/cc/lib/benchmark:sampler_benchmark --config=linux
-        bazel run -c opt src/cc/lib/benchmark:neighbor_sampler_benchmark --config=linux
+        bazel run -c opt src/cc/lib/benchmark:grpc_benchmark --config=linux --remote_cache=http://localhost:8080
```

Comment:
Are these supposed to be localhost, it looks different from the other ones?

Reply:
Yes, others also have a proxy set on local host. We don't use an env variable here, because this action only runs on linux and argument is explicit.